site/app/templates/polls/NewPollPage.twig

```diff
@@ -110,10 +110,10 @@
                         <textarea aria-label="Reponse text" class="poll_response" name="response_{{response}}"style="resize: none; min-height: 50px; max-height: 300px; height: 50px; width: 80%;"
                         rows="10" cols="30">{{poll.getResponseString(response)}}</textarea>
                         <div class="move-btn up-btn">
-                            <i class="move-arrow up"></i>
+                            <i class="fa fa-lg fa-chevron-up move-arrow up"></i>
```

Comment:
`move-arrow` and `up` are unused now, and can be removed:

```suggestion
                            <i class="fa fa-lg fa-chevron-up"></i>
```

Reply:
Be careful.  I tried removing them and the JS for moving the fields up and down broke.  This file could use some serious refactoring but as it stands now, the JS targets classes so removing them is a problem.  The classes are being used as IDs here.

Reply:
I double checked these suggestions and nothing seemed broken? Only the `up-btn`, `down-btn`, and `delete-btn` classes are bound here. `move-arrow` is not referenced anywhere outside of these class definitions, and likewise the other classes.

Reply:
Ok, looks good.